js/ui/context_menu/ui.menu_base.js

```diff
@@ -758,6 +758,10 @@ var MenuBase = HierarchicalCollectionWidget.inherit({
             case "useInkRipple":
                 this._invalidate();
                 break;
+            case "items":
```

Comment:
Check a case when items were changed in angular binding or data source changing

Reply:
In dataSource case optionChange for items is called. 
Trying to reproduce test case in angular.

Reply:
I mean that it may be possible to create such case when items option will be called (binding or data source changing) but we don't expect to clear focusing